pom.xml

```diff
@@ -13,7 +13,7 @@
     <parent>
         <groupId>org.springframework.boot</groupId>
         <artifactId>spring-boot-starter-parent</artifactId>
-        <version>1.4.1.RELEASE</version>
+        <version>1.5.3.RELEASE</version>
         <relativePath/>
```

Comment:
This is an update of springboot version. It must have his own PR because is not related with the content of this.

Reply:
Because a non merged PR this was necessary (because an error by my fault) for making the project work